### PR TITLE
Add more tests for BQ to VCF

### PR DESCRIPTION
**Pull request by allieychen**
Add integration test cases for BQ to VCF.

Issues: [85](https://github.com/googlegenomics/gcp-variant-transforms/issues/85)
Tested: integration tests.

### CODE REVIEW

File: gcp_variant_transforms/testing/data/vcf/bq_to_vcf/expected_output/4_0.vcf [hunk 1]
```diff
@@ -0,0 +1,15 @@
+##INFO=<ID=NS,Number=1,Type=Integer,Description="Number of Samples With Data">
```
[the remark this answers is not in the document]
Comment (allieychen) [reply gcp_variant_transforms/testing/data/vcf/bq_to_vcf/expected_output/4_0.vcf]: Thanks, will do this in another PR.

**Comment by allieychen** [comment]
Thank you Asha on the review. PTAL :)
Sorry I should have a separate commit for the new changes. The modifications are renaming of the `.vcf` files and updating the `.json` files.

File: gcp_variant_transforms/testing/data/vcf/README.md [hunk 2]
```diff
@@ -49,3 +49,8 @@ modifications:
 2. The penultimate variant has replaced `HQ` with `FT` in the FORMAT, which is
 not defined in the header.
 3. The `POS` value is removed from the last variant.
+
+The folder `bq_to_vcf` is created to test BQ to VCF pipeline. In
```
Comment (arostamianfar) [inline gcp_variant_transforms/testing/data/vcf/README.md]: nit: s/BQ/BigQuery

**Comment by allieychen** [comment]
Thank you, Asha. I have updated the unique temp id in this PR.

**Pull request merged**